### PR TITLE
Update to version 4.0.3

### PR DESCRIPTION
**Pull request by jongund**
* Fixed bug in setting `menuTextColor` property.
* Updated documentation.
* Removed some unused class names from action menu items.

### CODE REVIEW

File: cdn-info.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+# CDN Configuration Information and Copy Commands
```
Comment (brianteeman) [inline cdn-info.md:2]: These instructions look very specific to you university

File: src/css/SkipTo.template.css [hunk 2]
```diff
@@ -122,7 +122,7 @@
 .skip-to [role="menuitem"].skip-to-nesting-level-3 .label { grid-column: 4 / 8; }
 
 .skip-to [role="menuitem"].no-items .label,
-.skip-to [role="menuitem"].action .label {
+.skip-to [role="menuitem"].skip-to-action .label {
```
Comment (brianteeman) [inline src/css/SkipTo.template.css:125]: you cant just rename these things in a patch release. you are potentially breaking the markup of anyone who is using there own css. unless you are saying that the current class doesnt work

File: src/css/SkipTo.template.css [hunk 3]
```diff
@@ -59,7 +59,7 @@
   width: auto;
   border-width: 0px;
   border-style: solid;
-  color: $menuitemTextColor;
+  color: $menuTextColor;
```
Comment (brianteeman) [inline src/css/SkipTo.template.css:62]: isnt this supposed to be $menuitemColor

**Comment by jongund** [comment]
This is fixed in version 4.0.3, there is a pull request waiting

**Comment by brianteeman** [comment]
@jongund very confused. This is the only pull request

**Comment by jongund** [comment]
Found a bug that needed fixing.

**Pull request closed without merging**